lib/ProductOpener/Users.pm

```diff
@@ -426,90 +464,146 @@ sub check_user_form ($type, $user_ref, $errors_ref) {
+	}
 
-		my $mail = '';
-		process_template("emails/user_new_pro_account.tt.txt", $template_data_ref, \$mail);
+	if (not $org_created) {
+		# The requested org already exists
+		# build second part of the mail about it and alter the subject
+		$mail = '';
+		process_template("emails/user_new_pro_account_org_request_validated.tt.txt", $template_data_ref, \$mail);
 		if ($mail =~ /^\s*Subject:\s*(.*)\n/im) {
 			my $subject = $1;
 			my $body = $';
 			$body =~ s/^\n+//;
-			$template_data_ref->{mail_subject_new_pro_account} = URI::Escape::XS::encodeURIComponent($subject);
-			$template_data_ref->{mail_body_new_pro_account} = URI::Escape::XS::encodeURIComponent($body);
+			$template_data_ref->{mail_subject_new_pro_account_org_request_validated}
+				= URI::Escape::XS::encodeURIComponent($subject);
+			$template_data_ref->{mail_body_new_pro_account_org_request_validated}
+				= URI::Escape::XS::encodeURIComponent($body);
 		}
 		else {
-			send_email_to_producers_admin("Error - broken template: emails/user_new_pro_account.tt.txt",
+			send_email_to_producers_admin(
+				"Error - broken template: emails/user_new_pro_account_org_request_validated.tt.txt",
 				"Missing Subject line:\n\n" . $mail);
 		}
+	}
 
-		if (defined $requested_org_ref) {
-
-			# The requested org already exists
-			$mail = '';
-			process_template("emails/user_new_pro_account_org_request_validated.tt.txt", $template_data_ref, \$mail);
-			if ($mail =~ /^\s*Subject:\s*(.*)\n/im) {
-				my $subject = $1;
-				my $body = $';
-				$body =~ s/^\n+//;
-				$template_data_ref->{mail_subject_new_pro_account_org_request_validated}
-					= URI::Escape::XS::encodeURIComponent($subject);
-				$template_data_ref->{mail_body_new_pro_account_org_request_validated}
-					= URI::Escape::XS::encodeURIComponent($body);
-			}
-			else {
-				send_email_to_producers_admin(
-					"Error - broken template: emails/user_new_pro_account_org_request_validated.tt.txt",
-					"Missing Subject line:\n\n" . $mail);
-			}
-		}
-		else {
+	# Send an e-mail notification to admins, with links to the organization
+	$mail = '';
+	process_template("emails/user_new_pro_account_admin_notification.tt.html", $template_data_ref, \$mail);
+	if ($mail =~ /^\s*Subject:\s*(.*)\n/im) {
+		my $subject = $1;
+		my $body = $';
+		$body =~ s/^\n+//;
+		send_email_to_producers_admin($subject, $body);
+	}
+	else {
+		send_email_to_producers_admin("Error - broken template: emails/user_new_pro_account_admin_notification.tt.html",
+			"Missing Subject line:\n\n" . $mail);
+	}
+	return;
+}
 
-			# The requested org does not exist, create it
-			my $org_ref = create_org($userid, $user_ref->{requested_org});
-			add_user_to_org($org_ref, $userid, ["admins", "members"]);
+=head2 process_user_requested_org($user_ref)
 
-			$user_ref->{org} = $user_ref->{requested_org_id};
-			$user_ref->{org_id} = get_string_id_for_lang("no_language", $user_ref->{org});
+A user requested to be part of a producer organization.
```

Comment:
Is that the email sent ? It's a bit crude

Reply:
Not sure which mail you are talking about.

To the user, the only mail sent are:
- add_user_email_body (see common.po)
- reset_password_email_body (see common.po)

All the other mail are sent to the team.